CODEOWNERS

```diff
@@ -110,14 +110,8 @@ test/test_mps.py @kulinseth
 aten/src/ATen/mps/ @kulinseth
 aten/src/ATen/native/mps/ @kulinseth
 
-# Profiler
-torch/csrc/autograd/profiler* @robieta
-torch/autograd/profiler* @robieta
-torch/csrc/profiler/ @robieta
-torch/profiler/ @robieta
-
 # AOTDispatch tests
 test/functorch/test_aotdispatch.py @ezyang @Chillee
 
 # Dataloader
-torch/utils/data/ @NivekT @ejguan
```

Comment:
Why do you remove Kevin here?

Reply:
"Unknown owner on line 123: make sure @NivekT exists and has write access to the repository"